backend/.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow runs ESLint, Prettier, unit tests, and e2e tests using pnpm.
+name: CI
+
+on:
+  push:
+    branches:
+      - '!main'
+      - '!develop'
+      - '*'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js 16
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install dependencies
+        run: pnpm install
+      - name: Run unit tests
+        run: pnpm run test
+      - name: Run e2e tests
+        run: pnpm run test:e2e
+
+.husky/pre-commit:
```

Comment:
This looks out of its place. Generally, we follow prettier lint -> eslint -> unit test -> e2e tests

Reply:
Resolved